app/main.py

```diff
@@ -1,13 +1,27 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+def format_linter_error(error: dict[str, str]) -> dict[str, str]:
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+    }
 
 
-def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+def format_single_linter_file(file_path: str, errors: list[dict[str, str]])\
```

Comment:
Do not use `\` to split lines, it is a bad practice. Place each argument in a separate line instead